semana25/rodada-cases-semana25/casePokemonGo/src/index.ts

```diff
@@ -0,0 +1,19 @@
+import { Request, Response } from "express";
+import {app} from "./controller/app";
+
+import { generationPokemonCount } from "./endpoints/generationPokemonCount";
+import { generationSearch } from "./endpoints/genarationSearch";
+import { searchName } from "./endpoints/searchName";
+import { searchTypes } from "./endpoints/searchTypes";
+import { searchSpecific } from "./endpoints/searchSpecific";
+import { searchStatTotal } from "./endpoints/searchStatTotal";
+
+
+app.get("/pokemon/count", generationPokemonCount)
+app.get("/pokemon/:generation", generationSearch)
+app.get("/pokemon/name", searchName)
+app.get("/pokemon/types", searchTypes)
+app.get("/pokemon/specificType", searchSpecific)
+app.get("/pokemon/statTotal", searchStatTotal)
```

Comment:
É melhor deixar os endpoints com pathParams no final
```suggestion
app.get("/pokemon/count", generationPokemonCount)
app.get("/pokemon/name", searchName)
app.get("/pokemon/types", searchTypes)
app.get("/pokemon/specificType", searchSpecific)
app.get("/pokemon/statTotal", searchStatTotal)
app.get("/pokemon/:generation", generationSearch)
```